portal/flatpak-portal.c

```diff
@@ -777,6 +1295,7 @@ on_bus_acquired (GDBusConnection *connection,
   portal_flatpak_set_version (PORTAL_FLATPAK (portal), 1);
   g_signal_connect (portal, "handle-spawn", G_CALLBACK (handle_spawn), NULL);
   g_signal_connect (portal, "handle-spawn-signal", G_CALLBACK (handle_spawn_signal), NULL);
+  g_signal_connect (portal, "handle-create-update-monitor", G_CALLBACK (handle_create_update_monitor), NULL);
```

Comment:
What maps the `CreateUpdateMonitor` method to the `handle-create-update-monitor` signal? I guess it's automatic?

Reply:
This is handled by the gdbus code generation tool (i.e. it maps the method calls to similarly named signals).